pkg/kubelet/networks.go

```diff
@@ -54,6 +55,8 @@ func (nh *networkHost) SupportsLegacyFeatures() bool {
 // methods, because networkHost is slated for deletion.
 type criNetworkHost struct {
 	*networkHost
+	// criNetworkHost currently support legacy features. Hence no need to support PortMappingGetter
```

Comment:
The naming is confusing and I can't seem to remember what it's used for. Should probably clarify this in the comment or just rename it. Not a blocker for this PR though.